submissions/Welcome_to_GryphHacks /README.md

```diff
@@ -0,0 +1,26 @@
+# Hello World
+
+This is welcome to gryphhacks 2023 program.
+It is a simple executable python3 script that will print `Welcome to GryphHacks 2023` when run.
+
+## Usage
+
+To use this program you must have `python3` installed.
+
+Once python is installed the following commands can be used to run it.
+
+```bash
+$ cd submissions/welcome_to_gryphhacks
+```
+
+```bash
+$ ./welcome_to_gryphhacks.py
+```
+
+## Why I Added it
+
+This program is being added as an sample contribution.
+
+## Contributors
+
+This program was created by [Shreyas Jeevan Prakash].
```

Comment:
Nit: instead of `[Shreyas Jeevan Prakash]` try `[Shreyas Jeevan Prakash](https://github.com/ShreyasJ1P)` so you can add a link to your profile